src/pegout/components/RbtcInputAmount.vue

```diff
@@ -97,19 +97,23 @@ export default defineComponent({
         && amount.lte(balance);
     };
 
+    function emitGetQuotes() {
+      const weiBigAmount = new WeiBig(rbtcAmount.value, 'rbtc');
+      setRbtcAmount(weiBigAmount);
+      addAmount(weiBigAmount)
+        .then(() => calculateFee());
+      context.emit('valid-amount', isValidAmount(weiBigAmount), rbtcAmount.value);
+    }
+
+    const timeOutId = ref(0);
     const rbtcAmountModel = computed({
       get() {
         return rbtcAmount.value;
       },
       set(amount: string) {
+        window.clearTimeout(timeOutId.value);
```

Comment:
Adding comment from @alexjavabraz >> We really need to use the window object here?

Reply:
Not strictly necessary since it's global, but using window improves TypeScript inference.